locales/locales.php

```diff
@@ -1666,8 +1666,8 @@ public function __construct() {
 		$pirate->english_name = 'English (Pirate)';
 		$pirate->native_name = 'English (Pirate)';
 		$pirate->lang_code_iso_639_2 = 'art';
-		$pirate->wp_locale = 'art_xpirate';
-		$pirate->slug = 'art-xpirate';
+		$pirate->wp_locale = 'xx_pirate';
+		$pirate->slug = 'xx-pirate';
```

Comment:
Only the slug needs to be changed, the WP locale should remain as is.

xx-pirate looks odd, let’s just use pirate.